docs/protocol/on-chain/channel-lifecycle.mdx

```diff
@@ -0,0 +1,350 @@
+    Contract->>Contract: Verify CHANCLOSE magic
+    Contract->>Contract: Distribute funds
+    Contract->>Contract: Set status to FINAL
+    Contract->>Contract: Delete metadata
+    Contract->>User: Emit Closed event
+```
+
+:::success Preferred Method
+**This is the preferred closure method as it is fast and gas-efficient.** It requires only one transaction and completes immediately without a challenge period.
+:::
+
+## Closure - Challenge-Response
+
+**Purpose**: Handle closure when participants disagree or one party is unresponsive.
+
+### Challenge Process
+
+1. To initiate a challenge, a participant MAY:
+   - Call the `challenge` function with their latest valid state and required proofs
+
+2. The contract MUST:
+   - Verify the submitted state via the adjudicator
+   - If valid, store the state and start the challenge period
+   - Set a challenge expiration timestamp (current time + challenge duration)
+   - Set the channel status to `DISPUTE`
+   - Emit a `Challenged` event with channelId and expiration time
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant Contract
+    participant Timer
+    
+    User->>Contract: challenge(channelId, state, proofs)
+    Contract->>Contract: Verify state
+    Contract->>Contract: Store state
+    Contract->>Contract: Set status to DISPUTE
+    Contract->>Timer: Start challenge period
+    Contract->>User: Emit Challenged event
+```
+
+### Response Process
+
+During the challenge period, any participant MAY:
+- Submit a more recent valid state by calling `challenge` again
+- If the new state is valid and more recent (as determined by the adjudicator or IComparable interface), the contract MUST update the stored state and reset the challenge period
+
+```mermaid
+graph TB
+    A[Challenge Initiated] --> B{Response Received?}
+    B -->|Yes - Newer State| C[Update State]
+    C --> D[Reset Timer]
+    D --> B
+    B -->|No - Timer Expires| E[Proceed to Close]
+    
+    style A fill:#FFB6C1
+    style C fill:#87CEEB
+    style E fill:#90EE90
+```
+
+### Resolution
+
+After the challenge period expires, any participant MAY:
+- Call `close` to distribute funds according to the last valid challenged state
+
+The contract MUST:
+- Verify the challenge period has elapsed
+- Distribute funds according to the challenged state's allocations
+- Set channel status to `FINAL`
+- Delete the channel metadata
+- Emit a `Closed` event
+
+:::warning Key Principle
+The challenge mechanism gives parties time to prove they have a newer state. If no one responds with a newer state, the challenged state is assumed correct.
+:::
+
+**Complete Challenge-Response Flow**:
+
+```mermaid
+stateDiagram-v2
+    [*] --> Active
+    Active --> Dispute: challenge()
+    Dispute --> Dispute: challenge() with newer state
+    Dispute --> Final: close() after timeout
+    Final --> [*]
+    
+    note right of Dispute: Challenge period active<br/>Parties can submit<br/>newer states
+```
+
+## Reset Protocol (Resize)
```

Comment:
This section should be rewritten as it's not how it works)

Reply:
Apologies. I referred to incorrect markdown. I have corrected and added it.